docker/plugin-updates/src/main/scala/uk/gov/nationalarchives/pluginupdates/Decoders.scala

```diff
@@ -0,0 +1,36 @@
+package uk.gov.nationalarchives.pluginupdates
+
+import io.circe.{Decoder, HCursor}
+
+object Decoders {
+
+  case class PluginUpdates(pluginName: String, oldVersion: String, newVersion: String, warning: List[PluginWarning])
+
+  case class Plugin(name: String, version: String)
+
+  case class PluginVersions(pattern: String)
+
+  case class PluginWarning(message: String, name: String, versionPatterns: List[PluginVersions])
+
+  case class JenkinsPlugins(warnings: List[PluginWarning])
+
+  implicit val decodePluginVersion: Decoder[PluginVersions] = (c: HCursor) => for {
```

Comment:
Why do some of the case classes need decoders and some don't? Especially when they're as simple as `PluginVersions`, which contains a single string?

Reply:
I've just realised that none of them need decoders. The original reason I had it was because we were only using a subset of the json returned from the jenkins plugin endpoints and I thought you would have to create a case class with every field to keep circe happy but if you use the extras package, it will just ignore any json you haven't configured so that's cool. Anyway, I've removed the decoders class.

Reply:
Nice!